redhat-developer/developers.redhat.com
# [RHDX-117] Display Target Product on Video Resource as a Card

### JIRA Issue Link
[RHDX-117](https://projects.engineering.redhat.com/browse/RHDX-117)

### Verification Process
Some Video Resources have a value for Target Product.

Verify the Target Product displays as a Card on the Video Resource full view mode, such as 

https://developer-preview-3463.ext.us-west.dc.preprod.paas.redhat.com/videos/youtube/6EivZoILNFg/
https://developer-preview-3463.ext.us-west.dc.preprod.paas.redhat.com/videos/vimeo/39607154/


To achieve this, we had to:
- Enable the Card view mode for the Product content type.  Replicate the Featured Tile view mode, as it is used on the /products/ page.
- Use the Product Logo (field_image) in the card display.
- Update the Video Resource Default view mode to render the target product as a Card.
- Add twig templates

**Notes:**
- At this time, 17 products do not have logos, and will need to verify the card renders as expected when there is no image to display. See sheet linked to the JIRA issue.
- 692 videos do not have a target product, and should display the description in the full 12 column grid.
